### PR TITLE
tests: Add docker tag

### LINKED ISSUE #292: add docker tag
add integration docker tag test

### PR DESCRIPTION
**Pull request by GabyCT**
Add integration docker tag test

Fixes #292

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/tag_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		Expect(ContainerRemove(id)).To(BeTrue())
+		Expect(ContainerExists(id)).NotTo(BeTrue())
+		args = []string{"rmi", tagName}
+		runCommand(args, 0)
+	})
+
+	Describe("tag with docker", func() {
+		Context("tag a container", func() {
+			It("has the tag", func() {
+				tagName = "container"
```
Comment (devimc) [inline integration/docker/tag_test.go]: this tagName can clash with already existing tag names, could you use ```RandID```?

File: integration/docker/tag_test.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+var _ = Describe("tag", func() {
+	var (
+		args     []string
+		id       string
+		exitCode int
+		tagName  string
+		command  *Command
+	)
+
+	runCommand := func(args []string, expectedExitCode int) {
```
Comment (devimc) [inline integration/docker/tag_test.go]: I noticed that you are using a similar pattern in other tests, would be nice to include it in a common file

File: integration/docker/tag_test.go (hunk 1 above)
Comment (GabyCT) [inline integration/docker/tag_test.go]: @devimc the name does not accept numbers and that is what randid does, so I decided to put a name

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by devimc** [comment]
lgtm

**Pull request merged**